.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,20 @@
+# This file was auto-generated by the Firebase CLI
```

Comment:
Its better to separate context of files into separate PRs 
e.g.
PR for firebase addition
PR for refactoring
PR for styling
and so on, but  its just a hint